frontend/web/src/features/hostdashboard/HostFinanceTab.js

```diff
@@ -293,174 +439,255 @@ export default function HostFinanceTab() {
-                      <span>Amount</span>
+                <div className="payouts-section">
+                  <h3>Recent Payouts</h3>
+                  {loadingStates.payouts ? (
+                    <div style={{ padding: 12 }}>
+                      <ClipLoader size={28} loading />
                     </div>
-                    <div className="balance-divider" />
-
-                    <div className="balance-item">
-                      <div className="balance-left">
-                        <span className="balance-dot balance-dot--incoming" />
-                        <span className="balance-label">Incoming</span>
-                      </div>
-                      <div className="balance-amount">
-                        {formatMoney(balanceView.incomingTotal, balanceView.currency)}
+                  ) : payouts.length > 0 ? (
+                    <>
+                      <div className="table-wrap">
+                        <table className="payout-table">
+                          <thead>
+                            <tr>
+                              <th>Payout date</th>
+                              <th>Amount</th>
+                              <th>Status</th>
+                              <th>Payout ID</th>
+                            </tr>
+                          </thead>
+                          <tbody>
+                            {pageSlice(payouts, payoutsPage).map((payout) => (
+                              <tr key={payout.id || `${payout.arrivalDate}-${payout.amount}`}>
+                                <td>{payout.arrivalDate}</td>
+                                <td>{formatMoney(payout.amount, payout.currency)}</td>
+                                <td>
+                                  <StatusBadge status={payout.status} />
+                                </td>
+                                <td title={payout.id || ""}>{payout.id || " - "}</td>
+                              </tr>
+                            ))}
+                          </tbody>
+                        </table>
                       </div>
+
+                      <TablePager page={payoutsPage} setPage={setPayoutsPage} totalPages={payoutsTotalPages} />
+                    </>
+                  ) : (
+                    <p>No payouts found.</p>
+                  )}
+                </div>
+
+                <div className="payout-frequency">
+                  <h3>Payout Frequency</h3>
+
+                  <div className="pf-grid">
+                    <div className="pf-row">
+                      <label className="pf-label" htmlFor="pf-interval">
+                        Payout frequency
+                      </label>
+                      <select
+                        id="pf-interval"
+                        className="pf-select"
+                        value={payoutInterval ?? ""}
+                        onChange={(e) => {
+                          const v = e.target.value;
```

Comment:
here as well, avoid vague naming

```suggestion
                          const period = e.target.value;
```
> Don't forge to change all use instances